test/IntegrationTests/TicketService/test_delete_custom_field.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from http import HTTPStatus
+import requests
+import logging
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+class TestDeleteCustomFieldAPI(unittest.TestCase):
+    BASE_URL = "http://localhost:8000"
+
+    def setUp(self):
+        """Setup function to initialize valid customer GUID and custom field."""
+        logger.info("=== Initializing test setup ===")
+
```

Comment:
add a function name in setup and tear down function

def setUp(self):
        # Log the start of a test
        logger.info(f"Starting test: {self._testMethodName}")
    
    def tearDown(self):
        # Log the end of a test
        logger.info(f"Finished test: {self._testMethodName}")